stable/redis-ha/values.yaml

```diff
@@ -108,6 +135,9 @@ securityContext:
 ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
 ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature
 ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+nodeSelector:
```

Comment:
Let's not put any default `nodeSelector` values. Everyone has their own and will prevent scheduling if someone does not have these exact values on their nodes.